setup.py

```diff
@@ -484,6 +484,14 @@ def run(self):
     "torch_tensorrt.runtime": "py/torch_tensorrt/runtime",
 }
 
+if not PY_ONLY:
+    packages.append("torch_tensorrt.dynamo.runtime.meta_ops")
```

Comment:
@peri044 I believe we already have a section for stuff like this (search for torchscript related code)